crates/solvers/config/example.legacy.toml

```diff
@@ -0,0 +1,3 @@
+chain-id = "1"
+solver-name = "CoW Solver"
+endpoint = "https://solver.cow.fi"
```

Comment:
Should we append `/solve` to the URL? This is how it's done [here](https://github.com/cowprotocol/services/a87229fb2ce8062a6ce5a8e22bf12c7bb25bba4e/main/crates/solvers/src/tests/legacy/mod.rs) and tests fails without it.

Reply:
Appending `/solve` should not be needed. The test setup code you are referring to was written when the `DefaultHttpSolverApi` still used a hardcoded `/solve` endpoint. But this is no longer the case so we could drop it. We would just have to also slightly adjust the failing tests so they also no longer send the requests to `<URL>/solve`.